### DISCUSSION TITLE
테일윈드 css 내장 애니메이션

### DISCUSSION THREAD

**Discussion by wjsdncl** [Part3-Team15/taskify#142] [Show and tell]
https://onlydev.tistory.com/141

테일윈드에서 기본적으로 애니메이션을 제공하네요
스핀이랑 핑 같은거 사용하면 좋을거 같아요 👍

**Comment by un0211** [comment]
아까 댓글을 쓰다 말았네요!!
민재님 스켈레톤도 테일윈드 기본 애니메이션 적용한거였죠..?
테일윈드 좋은 기능이 많네요..ㅎㅎ

**Reply by wjsdncl** [reply]
테일윈드가 빠르게 프로젝트 만들때 좋은거 같아요 기본적으로 기능이 많고 플러그인도 많아서
